src/main/java/thePackmaster/cards/eurogamepack/Prosper.java

```diff
@@ -0,0 +1,34 @@
+package thePackmaster.cards.eurogamepack;
+
+
+import com.megacrit.cardcrawl.actions.watcher.SkipEnemiesTurnAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.SpireAnniversary5Mod.modID;
+
+public class Prosper extends AbstractVPCard{
+    public static final String ID = makeID("Prosper");
+
+    public Prosper() {
+        super(ID, 3, CardType.SKILL, CardRarity.RARE, CardTarget.SELF);
+        purgeOnUse = true;

```

Comment:
Why purge instead of exhaust?

Reply:
Balance wise, it isn't too difficult to get victory one a turn if you do it right, so i was worried about infinite turns with exhumes but it looks like there aren't too many unconditional exhumes so i might make it exhaust.